integration-tests/infinispan-client/pom.xml

```diff
@@ -76,6 +80,24 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+
+        <!-- We override these to satisfy Infinispan server -->
```

Comment:
Does this mean that it won't work with 2.9.10.20191020? IMHO the BOM should be updated to Jackson 2.10.0 first

Reply:
The problem is that Infinispan Server uses a new version of Jackson that isn't compatible with the version Quarkus is currently using. The Server is only started to test the client, so this doesn't affect any Quarkus code.

Reply:
But doesn't that means that in these tests the client will use Jackson 2.10.0 too?

Reply:
The tests don't use anything that requires Jackson to be loaded afaik.

Reply:
At least it doesn't fail 🤷‍♂

Reply:
good question.

Reply:
Actually, is the Infinispan client using Jackson at all? I guess if it's only used in the server, then it's ok.

Reply:
The Infinispan Client does not use Jackson at all, it is all hotrod. I am only overriding the Jackson version for the Infinispan Server (which does use Jackson for REST), so that the client can actually talk to something.

I was referring to whether or not the Quarkus web layer used Jackson, maybe it only uses it for non String types? All my return types are String in the test.